app/models/forms/bulk_upload_lettings/year.rb

```diff
@@ -0,0 +1,26 @@
+module Forms
+  module BulkUploadLettings
+    class Year
+      include ActiveModel::Model
+
+      attr_accessor :year
+
+      validates :year, presence: true
+
+      def view_path
+        "bulk_upload_lettings_logs/forms/year"
+      end
+
+      def options
+        [
+          OpenStruct.new(id: "2022", name: "2022/2023"),
+          OpenStruct.new(id: "2021", name: "2021/2022"),
```

Comment:
```suggestion
          OpenStruct.new(id: "2022", name: "2022/2023"),
          OpenStruct.new(id: "2021", name: "2021/2022"),
```

Ideally these are derived from `FormHandler` so we don't need to change them every year.

Reply:
I've updated this. Not sure if my implementation meets our needs?